contracts/collections/sg721-base/src/msg.rs

```diff
@@ -46,46 +66,58 @@ pub enum ExecuteMsg<T, E> {
     Extension { msg: E },
 }
 
+#[cw_ownable_query]
+#[derive(QueryResponses)]
 #[cw_serde]
 pub enum QueryMsg {
+    #[returns(OwnerOfResponse)]
     OwnerOf {
         token_id: String,
         include_expired: Option<bool>,
     },
+    #[returns(ApprovalResponse)]
     Approval {
         token_id: String,
         spender: String,
         include_expired: Option<bool>,
     },
+    #[returns(ApprovalsResponse)]
     Approvals {
         token_id: String,
         include_expired: Option<bool>,
     },
+    #[returns(OperatorsResponse)]
     AllOperators {
         owner: String,
         include_expired: Option<bool>,
         start_after: Option<String>,
         limit: Option<u32>,
     },
+    #[returns(NumTokensResponse)]
     NumTokens {},
+    #[returns(ContractInfoResponse)]
     ContractInfo {},
-    NftInfo {
-        token_id: String,
-    },
+    #[returns(NftInfoResponse<Empty>)]
```

Comment:
double check this works for sg721-metadata-onchain

Reply:
How to double check? Do you mean after merge?